sql/core/benchmarks/FilterPushdownBenchmark-results.txt

```diff
@@ -578,3 +578,127 @@ Native ORC Vectorized                       11622 / 12196          1.4         7
 Native ORC Vectorized (Pushdown)            11377 / 11654          1.4         723.3       1.0X
 
 
+================================================================================================
+Pushdown benchmark for Timestamp
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
+Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
+
+Select 1 timestamp stored as INT96 row (value = CAST(7864320 AS timestamp)): Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
```

Comment:
shall we add a new line after the benchmark name? e.g.
```
Select 1 timestamp stored as INT96 row (value = CAST(7864320 AS timestamp)):
Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
...
```

We can send a follow-up PR to fix this entire file.

Reply:
OK. I'll send a follow-up PR.